lib/url-sync.js

```diff
@@ -116,6 +123,7 @@ class URLSync {
   }
 
   render({helper}) {
+    // FIXME use state from parameters
```

Comment:
? Would better remove FIXME, because we will never solve it otherwise :) If it works now but somehow it could be better then > issue. Reduces FUD